### PR TITLE
PR initial draft of PPP

### PR DESCRIPTION
**Pull request by JingYenLoh**
### What this does
This PR adds my initial draft of my PPP, where I document my contributions to the project.

Part of #183.

### How to test

Help me proofread!

### Notes
I also add CodeCov badges to our site and README, as requested by @mkeoliya .

### CODE REVIEW

File: docs/team/jingyenloh.md [hunk 1]
```diff
@@ -3,44 +3,46 @@ layout: page
...
+  * Highlights: This enhancement affects existing allocations and models to be added in future, such as Bills. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required the Semester to be made Observable, rather than a POJO like the rest of the AddressBook/ResiReg models.
 
-* **New Feature**: Added a history command that allows the user to navigate to previous commands using up/down keys.
+* **New Feature**: Added `--allocated` and `--vacant` flags to the `rooms` command
+  * What it does: allows the user to optionally view only allocated or vacant rooms, versus the original command where it would display all the rooms.
+  * Justification: This allows the user more find grained viewing of the rooms, for when they only want to view rooms with students residing in them (e.g. for maintainence services), or when they only want to view vacant rooms (e.g. to see if last minute allocations are allowed for an Exchange student)
```
Comment (chrisgzf) [inline docs/team/jingyenloh.md:19]:
```suggestion
  * Justification: This allows the user more fine-grained viewing of the rooms, for when they only want to view rooms with students residing in them (e.g. for maintainence services), or when they only want to view vacant rooms (e.g. to see if last minute allocations are allowed for an Exchange student)
```
Can you proof-read your changes first before PR-ing? Please don't waste your reviewers' time.

File: docs/team/jingyenloh.md [hunk 2]
```diff
@@ -3,44 +3,46 @@ layout: page
 title: Loh Jing Yen's Project Portfolio Page
 ---
 
-## Project: AddressBook Level 3
+## Project: ResiReg
 
-AddressBook - Level 3 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+**ResiReg** is a desktop residential college management application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 20 kLoC.
```
Comment (chrisgzf) [inline docs/team/jingyenloh.md:8]: `residential college management application used for teaching Software Engineering principles`

Wow. How can a residential college management application be used for teaching Software Engineering principles? Please enlighten.

**Review by chrisgzf — CHANGES_REQUESTED** [review]
Oops accidentally added a review. How to delete?

EDIT: idk

File: docs/team/jingyenloh.md (hunk 2 above)
Comment (Lysire) [inline docs/team/jingyenloh.md]: I don't think the purpose of the application is quite right as the application is supposed to be used to manage a residential college and not teach Software Engineering principles, which is the purpose of AB3. I think that there might have been an oversight regarding this.

Comment (JingYenLoh) [reply docs/team/jingyenloh.md]: Thanks for this! I've remedied it in the latest commit.

**Review by Lysire — APPROVED** [review]
Thanks for the fixes. LGTM!

**Pull request merged**